pkgs/applications/video/filebot/default.nix

```diff
@@ -1,52 +1,60 @@
 { lib, stdenv, fetchurl, openjdk17, makeWrapper, autoPatchelfHook
-, zlib, libzen, libmediainfo, curlWithGnuTls, libmms, glib
+, buildFHSUserEnv, zlib, libzen, libmediainfo, curlWithGnuTls, libmms, glib
 }:
 
-stdenv.mkDerivation rec {
-  pname = "filebot";
-  version = "4.9.6";
+let
+  filebot = stdenv.mkDerivation rec {
+    pname = "filebot";
+    version = "4.9.6";
 
-  src = fetchurl {
-    url = "https://web.archive.org/web/20220305095926/https://get.filebot.net/filebot/FileBot_${version}/FileBot_${version}-portable.tar.xz";
-    sha256 = "sha256-3j0WmmamE9KUNwjOVZvrdFH5dS/9FHSdbLfcAsOzQOo=";
-  };
-
-  unpackPhase = "tar xvf $src";
+    src = fetchurl {
+      url = "https://web.archive.org/web/20220305095926/https://get.filebot.net/filebot/FileBot_${version}/FileBot_${version}-portable.tar.xz";
```

Comment:
The URL seems to be versioned. Do they delete old version or why to we se archive.org?

Reply:
This is from https://github.com/NixOS/nixpkgs/pull/122219. They seem to occasionally change the tarball without bumping the version number.